api/src/api.js

```diff
@@ -102,12 +154,12 @@ function getSource(referralString) {
 async function fetchLatestHandler (req, res, next) {
   const searchParams = req.query;
 
-  if (!searchParams.app_id) {
+  if (!searchParams.app_id) { //TODO: Edit proofreading to allow timestamp OR job hash
     return res.status(400).send({msg: config.errNoId});
-  } else if (!searchParams.from) {
+  } else if (!searchParams.from && !searchParams.token) {
```

Comment:
oh i see what you're saying, that if the user doesn't specify a value for `from` we should default to 0?

Reply:
That's an option, but what I was saying is that 0 should be a valid option. If the user says "from=0", it should return all events.

Reply:
(right now it errors because 0 is falsey so `!searchParams.from` will be `true` when `from=0`)

Reply:
That's odd, my tests in postman have always accepted 0 as a valid parameter and returned data

Reply:
Interesting... I haven't tried in a while, maybe something changed? I've left mine as "1" because of this. But if your tests work than it must be a string at the point of this check, so all good!